src/snowflake/snowpark/functions.py

```diff
@@ -896,13 +959,37 @@ def ceil(e: ColumnOrName) -> Column:
 
 
 def cos(e: ColumnOrName) -> Column:
-    """Computes the cosine of its argument; the argument should be expressed in radians."""
+    """Computes the cosine of its argument; the argument should be expressed in radians.
+
+    Example:
+
+        >>> df = session.create_dataframe([[30]], schema=["deg"])
```

Comment:
Let's try a value that gives an easier result so we don't assert at the 15th digit, say `0`, `math.pi` etc.